types/node/index.d.ts

```diff
@@ -3143,7 +3143,6 @@ declare module "fs" {
 
     export class ReadStream extends stream.Readable {
         close(): void;
-        destroy(): void;
```

Comment:
Why is `destrory()` removed here? According to docs it should be here https://nodejs.org/dist/latest-v10.x/docs/api/stream.html#stream_readable_destroy_error